index.bs

```diff
@@ -6529,8 +6517,30 @@ The main benefits offered to [=[WRPS]=] by this specification include:
 1. The [=[RP]=] does not need to store additional secrets in order to gain the above benefits.
 
 As stated in the [[#sctn-conforming-relying-parties|Conformance]] section, the [=[RP]=] MUST behave as described in [[#sctn-rp-operations]]
-to obtain all of the above security benefits. However, one notable use case that departs slightly from this is described in the
-next section.
+to obtain all of the above security benefits. However, one notable use case that departs slightly from this is described below in [[#sctn-attestation-limitations]].
+
+
+### Visibility Considerations for Embedded Usage ### {#sctn-seccons-visibility}
+
+Simplistic use of WebAuthn in an embedded context, e.g., within <{iframe}>s as described in [[#sctn-iframe-guidance]], may make users vulnerable to <dfn>UI Redressing</dfn> attacks, also known as "[Clickjacking](https://en.wikipedia.org/wiki/Clickjacking)". This is where an attacker overlays their own UI on top of a [=[RP]=]'s intended UI and attempts to trick the user into performing unintended actions with the [=[RP]=]. For example, using these techniques, an attacker might be able to trick users into purchasing items, transferring money, etc.
+
+Even though WebAuthn-specific UI is typically handled by the [=client platform=] and thus is not vulnerable to [=UI Redressing=], it is likely important for an [=[RP]=] having embedded WebAuthn-wielding content to ensure that their content's UI is visible to the user. An emerging means to do so is by observing the status of the experimental [Intersection Observer v2](https://w3c.github.io/IntersectionObserver/v2/)'s `isVisible` attribute. For example, the [=[RP]=]'s script running in the embedded context could pre-emptively load itself in a popup window if it detects `isVisble` being set to `false`, thus side-stepping any occlusion of their content.
```

Comment:
nit: I love the description of "WebAuthn-wielding", but it's probably overly colorful for the document. Perhaps consider being plainer with just "WebAuthn-using"?